src/components/Facepile/Facepile.Props.ts

```diff
@@ -8,6 +8,18 @@ export interface IFacepileProps extends React.Props<Facepile> {
    * is fixed at PersonaSize.extraSmall regardless of what is specified.
    */
   personas: IFacepilePersona[];
+
+  /** Maximum number of personas to show */
+  maxDisplayablePersonas?: number;
+
+  /** Show add person button */
+  showAddButton?: boolean;
```

Comment:
Is there a reason the Facepile control uses <button> tag to wrap the Persona components instead of using the React Button component?